functional-tests/bdd-tests/TestBaseNavigation.cs

```diff
@@ -85,34 +86,16 @@ public void ClickOnBrandingChangeLink(string changeType)
         [And(@"I click on the Licences tab")]
         public void ClickLicencesTab()
         {
-            // check for failure to load Licences & Authorizations tab successfully
-            for (int i = 0; i < 3; i++)
-            {
-                try
-                {
-                    ngDriver.Navigate().GoToUrl($"{baseUri}licences");
-
-                    if (ngDriver.FindElement(By.XPath("//body[contains(.,'Details')]")).Displayed == false)
```

Comment:
@msehudi - I was checking for the Details column header to make sure that the licence details were available. I don't think the header will differentiate between a successful/unsuccessful page load?